pkg/util/stringutil/string_util.go

```diff
@@ -228,53 +281,76 @@ func CompileLike2Regexp(str string) string {
 	return string(result)
 }
 
-// DoMatchBytes is a adapter for `DoMatchInner`, `str` can only be an ascii string.
+// DoMatchBytes is an adapter for `DoMatchInner`, `str` can only be an ascii string.
 func DoMatchBytes(str string, patChars, patTypes []byte) bool {
```

Comment:
Also, I think we don't need to distinguish `DoMatchBytes` and `DoMatchPureBytes`. We can directly modify the behavior of `DoMatchBytes` (under proper tests).

Reply:
Make sense, I'll try.